class.photon.php

```diff
@@ -388,6 +389,24 @@ public static function filter_the_galleries( $galleries ) {
 		return $galleries;
 	}
 
+
+	/**
+	 * Runs the image widget through photon.
+	 *
+	 * @param array $instance Image widget instance data.
+	 * @return array
+	 */
+	function filter_the_image_widget( $instance ) {
+		if ( Jetpack::is_module_active( 'photon' ) && ! $instance['attachment_id'] && $instance['url'] ) {
+			jetpack_photon_url( $instance['url'], array(
```

Comment:
Hm, looks like we don't have a filter that would let us have access to the `<img>` already built, only the instance filter which gives us URL or attachment ID. I've asked if they're open to adding a filter before it's merged, but assuming they don't, is this a reasonable approach?